lib/saveable_extraction.rb

```diff
@@ -85,7 +85,7 @@ def process_multiple_choice(choice_ids, rationales = {}, is_final = nil)
         logger.error("Missing Embeddable::MultipleChoiceChoice id: #{choice_ids.join(",")}")
       end
       if ! multiple_choice
-        logger.error("Missing Embeddable::MultipleChoice id: #{choice.multiple_choice_id}")
```

Comment:
Oh another point. If we are allowing LARA to send multiple choice answers that do not have any choices, then we should not log that exception. Seems the best option would be for the code to differentiate between an empty list of choice_ids and one that has choice_ids that could not be found.